ie/ie.go

```diff
@@ -13,288 +13,291 @@ import (
 	"github.com/wmnsk/go-pfcp/internal/utils"
 )
 
+//go:generate stringer -type=IEType
+type IEType uint16
```

Comment:
Doesn't defining this as an alias work?

```suggestion
type IEType = uint16
```